src/rest_vol.h

```diff
@@ -192,9 +192,79 @@
+                                        ERR_MINOR, ret_value)                                                \
+    do {                                                                                                     \
+        CURLcode result = curl_easy_perform(curl_ptr);                                                       \
+                                                                                                             \
+        /* Reset the cURL response buffer write position pointer */                                          \
+        local_response_buffer.curr_buf_ptr = local_response_buffer.buffer;                                   \
+                                                                                                             \
+        if (CURLE_OK != result)                                                                              \
+            FUNC_GOTO_ERROR(ERR_MAJOR, ERR_MINOR, ret_value, "%s", curl_easy_strerror(result));              \
+                                                                                                             \
+        if (handle_HTTP_response) {                                                                          \
+            long response_code;                                                                              \
+                                                                                                             \
+            if (CURLE_OK != curl_easy_getinfo(curl_ptr, CURLINFO_RESPONSE_CODE, &response_code))             \
+                FUNC_GOTO_ERROR(ERR_MAJOR, ERR_MINOR, ret_value, "can't get HTTP response code");            \
+                                                                                                             \
+            HANDLE_RESPONSE(response_code, ERR_MAJOR, ERR_MINOR, ret_value);                                 \
+        } /* end if */                                                                                       \
+    } while (0)
+
+/* Counterpart of CURL_PERFORM that takes a curl multi handle,
+ * and waits until all requests on it have finished before returning. */
+
+#define CURL_MULTI_PERFORM(curl_multi_ptr, ERROR_MAJOR, ERR_MINOR, ret_value)                                \
+    int      still_running = 0;                                                                              \
+    CURLMsg *curl_mult_msg = NULL;                                                                           \
+                                                                                                             \
+    do {                                                                                                     \
+        fprintf(stderr, "%zu transfers still running\n", still_running);                                     \
+        fprintf(stderr, "Performing...");                                                                    \
```

Comment:
It's fine to have these fprintf calls behind debug macros, but otherwise the REST VOL shouldn't print output at all except in cases where an error occurs and the error stack mechanism hasn't been disabled

Reply:
This is an old macro that was unused at the time - I've since moved the finished version under the macro and removed these debug prints.